src/Domain/Donation.php

```diff
@@ -827,15 +828,7 @@ private function getConfirmedChampionWithdrawalTotal(): string
             return '0.0';
         }
 
-        $withdrawalTotal = '0.0';
-        foreach ($this->fundingWithdrawals as $fundingWithdrawal) {
-            // Rely on Doctrine `SINGLE_TABLE` inheritance structure to derive the type from the concrete class.
-            if ($fundingWithdrawal->getCampaignFunding()->getFund() instanceof ChampionFund) {
-                $withdrawalTotal = bcadd($withdrawalTotal, $fundingWithdrawal->getAmount(), 2);
-            }
-        }
-
-        return $withdrawalTotal;
+        return $this->getWithdrawalTotalByFundType(ChampionFund::class);
```

Comment:
this is going to conflict with the PR I've just created (see my comment at https://github.com/thebiggive/matchbot/pull/1230/files#r1965884768 )